### PR TITLE
Review

### PR DESCRIPTION
**Pull request by AmirKremer**
This for allowing us to easily write comments, don't merge :P

### CODE REVIEW

File: test/config/deploy.rb [hunk 1]
```diff
@@ -1,12 +0,0 @@
-set :application, 'capistrano-rails-artifact'
-
-#The Vagrantfile used python to start a simple server serving contents of /vagrant/test (which is actually this project's test directory)
-set :rails_artifact_archive_location, 'http://127.0.0.1:8080/test-build-artifact.tar.gz'
-
-set :rails_artifact_group, 'rails_runners'
```
Comment (AmirKremer) [inline test/config/deploy.rb:6]: Is this still relevant?

File: lib/capistrano/tasks/rails_artifact.rake [hunk 2]
```diff
@@ -1,26 +0,0 @@
...
-  desc 'Copy repo to releases'
-  task :create_release do
-    on release_roles :all do
-      archive_url = fetch(:rails_artifact_archive_location)
-      file_location = '/tmp/build-artifact-files/build-artifact.tar.gz'
-      execute :mkdir, '-p', File.dirname(file_location)
-      execute :rm, "-f #{file_location}"
-
-      execute :mkdir, '-p', release_path
-      execute :wget, "--no-check-certificate  -q -O '#{file_location}' '#{archive_url}'"
```
Comment (AmirKremer) [inline lib/capistrano/tasks/rails_artifact.rake:15]: Do we still want the no-check-certificate?

File: lib/capistrano/rails_artifact.rb [hunk 3]
```diff
@@ -1,13 +0,0 @@
-# This file will be included by default when someone sets the scm to be :rails_artifact
-
-load File.expand_path('../tasks/rails_artifact.rake', __FILE__)
-
-# These two tasks are unnecessary for this strategy
```
Comment (AmirKremer) [inline lib/capistrano/rails_artifact.rb:5]: Maybe add something about it in the Readme?

File: lib/capistrano/tasks/rails_artifact.rake [hunk 4]
```diff
@@ -1,26 +0,0 @@
-namespace :rails_artifact do
-  desc 'Check that the repo is reachable'
-  task :check do
-  end
-
-  desc 'Copy repo to releases'
-  task :create_release do
-    on release_roles :all do
```
Comment (AmirKremer) [inline lib/capistrano/tasks/rails_artifact.rake:8]: Is there anything you need to know regarding permissions? Should it be added to the Readme?

File: test/capistrano-rails-artifact_test.rb [hunk 5]
```diff
@@ -1,83 +0,0 @@
...
-require 'sshkit/dsl'
-
-class CapistranoRailsArtifactSpec < MiniTest::Unit::TestCase
-  def setup
-    system("vagrant up")
-    configure_sshkit_for_vagrant
-  end
-
-  def teardown
-    #system("vagrant destroy -f")
```
Comment (AmirKremer) [inline test/capistrano-rails-artifact_test.rb:12]: I am guessing this should happen by default and adding a param to override it?

File: README.md [hunk 6]
```diff
@@ -1,63 +0,0 @@
...
-
-This gem assumes that you have a tar.gz file in an accessible place. This .tar.gz should contain all the gems
-vendored into vendor/bundle as well as the compiled assets. An example command to build this archive is:
-
-```sh
-export RAILS_ENV=production
-rm -rf vendor/bundle
-bundle install --without development test --deployment
-bundle exec rake assets:precompile
-echo `git rev-parse HEAD` > REVISION
```
Comment (AmirKremer) [inline README.md:32]: What is this line for?

File: test/config/deploy.rb (hunk 1 above)
Comment (ctaintor) [inline test/config/deploy.rb:6]: yes

File: lib/capistrano/tasks/rails_artifact.rake (hunk 2 above)
Comment (ctaintor) [inline lib/capistrano/tasks/rails_artifact.rake:15]: perhaps that should be configurable... but, yea - probably shouldn't be the default

File: lib/capistrano/rails_artifact.rb (hunk 3 above)
Comment (ctaintor) [inline lib/capistrano/rails_artifact.rb:5]: you mean... saying that you don't need to do anything but add the scm :rails_artifact?

File: lib/capistrano/tasks/rails_artifact.rake (hunk 4 above)
Comment (ctaintor) [inline lib/capistrano/tasks/rails_artifact.rake:8]: I don't think there's anything unique

File: test/capistrano-rails-artifact_test.rb (hunk 5 above)
Comment (ctaintor) [inline test/capistrano-rails-artifact_test.rb:12]: didn't mean to have it uncommented. do you think a param is necessary?

File: README.md (hunk 6 above)
Comment (ctaintor) [inline README.md:32]: I'll add a comment - adds the git revision SHA-1 to a file named `REVISION`

File: lib/capistrano/rails_artifact.rb (hunk 3 above)
Comment (AmirKremer) [inline lib/capistrano/rails_artifact.rb:5]: Tasks ;)

**Pull request closed without merging**